client/src/components/public/calendar/EventType.vue

```diff
@@ -302,19 +301,28 @@ export default {
     },
     methods: {
         ...mapActions('publicEvent', {
-            getEventTypeById: actions.GET_EVENT_TYPE_BY_ID
-        }),
-        ...mapMutations('publicEvent', {
-            setPublicEvent: mutations.SET_PUBLIC_EVENT
+            getEventTypeById: actions.GET_EVENT_TYPE_BY_ID,
+            setPublicEvent: actions.SET_PUBLIC_EVENT
         }),
+        getStartDate(time) {
+            return `${momentTimezones(
+                `${this.date} ${time}`,
+                'YYYY-MM-DD HH:mm'
```

Comment:
HH:mm:ss

Reply:
I think this date shouldn't contain seconds and it would be better if there were only hours and minutes, as these seconds will not be used anywhere, it's NOT data that will be written to database